src/components/InterestForm/index.tsx

```diff
@@ -28,89 +30,110 @@ export default function InterestForm({ caseData }: { caseData: CaseListing }) {
         listing_id: caseData.id,
         listing_type: 'CASE',
-        user_id: '36b8f84d-df4e-4d49-b662-bcde71a8764f',
+        user_id: '515d9b33-9185-4601-9a13-23789bea3ac0',
         form_response: {
+          start_date: new Date(startDate),
+          interestReason: reason,
           rolesInterested:
             rolesInterested === radioOptions[2]
               ? ['ATTORNEY', 'INTERPRETER']
               : [rolesInterested.toUpperCase() as RoleEnum],
-          interestReason: reason,
-          start_date: new Date(startDate),
         },
       };
 
-      await insertInterest(newInterest);
+      await upsertInterest(newInterest);
       setReason('');
       setStartDate('');
       setRolesInterested('');
+      setSubmitted(true);
     }
   };
 
   return (
     <FormContainer>
-      <FormTitle>Submit Interest</FormTitle>
-      <H4>What role(s) are you applying for?</H4>
-      <RadioGroup>
-        {radioOptions.map(option => (
-          <RadioLabel key={option}>
-            <RadioInput
-              id="radio"
-              type="radio"
-              name="radioOptions"
-              value={option}
-              checked={rolesInterested === option}
-              onChange={event => setRolesInterested(event.target.value)}
-            />
-            {option}
-          </RadioLabel>
-        ))}
-      </RadioGroup>
-
-      <H4>What is the earliest date you can contact the client?</H4>
-      <FormInput
-        id="startDate"
-        required
-        placeholder="MM/DD/YYYY"
-        value={startDate}
-        onChange={event => setStartDate(event.target.value)}
-      />
-
-      <H4>Why are you interested in this case?</H4>
-      <FormTextArea
-        id="reason"
-        required
-        value={reason}
-        onChange={event => setReason(event.target.value)}
-      />
-      <FormFooter>
-        <FormWarning>
-          Your interest form is not saved!
-          <br />
-          Please submit before leaving this page.
-        </FormWarning>
-        <Button
-          primaryColor={COLORS.blueMid}
-          secondaryColor={COLORS.blueDark}
-          onClick={handleInsert}
-        >
-          Submit Interest
-        </Button>
-      </FormFooter>
+      <H3>Submit Interest</H3>
+      {submitted ? (
+        <P>We have received your submission!</P>
+      ) : (
+        <>
+          <FormQuestion $color={COLORS.greyDark}>
+            What role(s) are you applying for?
+          </FormQuestion>
+          <RadioGroup>
+            {radioOptions.map(option => (
+              <Radio key={option}>
+                <RadioInput
+                  id={`radio${option}`}
+                  type="radio"
+                  name="radioOptions"
+                  value={option}
+                  checked={rolesInterested === option}
+                  onChange={event => setRolesInterested(event.target.value)}
+                />
+                <RadioLabel>{option}</RadioLabel>
```

Comment:
I'd recommend adding a `htmlFor` prop to the labels that match the `id` of the `RadioInput` they correspond to, which will let you select the radio when you click on the label.